### DISCUSSION TITLE
Add the ability to search starred 🌟 repos list

### DISCUSSION THREAD

**Discussion by Bad3r** [community/community#4695] [Mobile]
It would great if we can filter the list of stared repos or sort them alphabetically

**Comment by brianlovin** [comment]
Thanks for the suggestion @Bad3r – we agree! We're planning to do some work to improve the entire starring flow on mobile in the coming months.

**Reply by Bad3r** [reply]
@brianlovin hey Brian. It's been more than two years and there is no updates yet regarding this feature.

**Reply by mpepping** [reply]
Searching trough Starred repos is definitely a feature that adds value in the app.
